addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -18,13 +18,13 @@
         <a t-if="has_button_access"
                 t-att-href="button_access['url']"
                 style="padding: 8px 12px; font-size: 12px; color: #FFFFFF; text-decoration: none !important; font-weight: 400; background-color: #875A7B; border: 0px solid #875A7B; border-radius:3px">
-            <t t-esc="button_access['title']"/>
+            <t t-out="button_access['title']"/>
```

Comment:
in your commit message, no need to write the filepath

Maybe just something like this ?
> Replace t-esc with t-out in email templates layout